packages/pds-ember/addon/components/pds/breadcrumbs/index.hbs

```diff
@@ -1,7 +1,6 @@
 <nav
   class="pds-breadcrumbs"
   aria-label={{t 'pds.components.breadcrumbs.aria.label'}}
-  role="navigation"
```

Comment:
new template lint rules called this out as redundant on account of this already being a `<nav>`